blog/designing-for-journey-candance-van-oostrum.md

```diff
@@ -0,0 +1,35 @@
+---
+slug: designing-for-journey-candance-van-oostrum
+title: Designing for the Journey
+author: Candace Van Oostrum
+author_title: First Designer
+author_image_url: https://media-exp1.licdn.com/dms/image/C5603AQFq2qVIOpFU5Q/profile-displayphoto-shrink_200_200/0/1611000515802?e=1656547200&v=beta&t=xu184mRNVPc1qHTvHoZ3LE_udGzYG5_l2GtXf34kiD0
+tags:
+  - reflections
+date: 2022-4-27T00:00:00Z
+image: /img/an-error-occured.png
+---
+
+![An error coccured image](/img/an-error-occured.png)
+
+<!-- truncate -->
+
+Why would a highly technical and backend engineering focused company even need a designer?
+
+When I first started my design career, there was no YouTube, Twitter or responsive design and frankly, designers and developers didn’t talk. Designers and developers were on different floors, in different buildings, and in different cities. This was quite problematic and co-locating teams became the norm to force and foster collaboration.
+
+In my earlier days I used to be insecure about being the only woman at the table and believed that I had to be territorial about my UX title. When engineers shared their opinions on what the user experience should be, I regrettably didn’t take their input very generously. I thought it was my job to be the expert and the sole person with an opinion on the UX.  Over the past 20 years I’ve had the privilege of working with some truly amazing engineers, solution architects, scrum masters, engineering managers, you name it. From those working experiences, I’ve really come to depend on my engineering counterparts to help shape and realize the UX vision.
+
+This personal transformation took time. There were bumps, and I learned some really valuable lessons the hard way. The most important lesson I learned was how to build meaningful relationships across the aisle. I can say with a high degree of confidence that I owe a great part of my success to collaborating, pairing and partnering with engineers.  
+
+What I love most about my job is that product designers cut across entire organizations. The success of the role is largely dependent on building trust and establishing true partnerships with key stakeholders. Design interfaces with growth marketing, dev rel, product, engineering and customer success. This gives us a really unique vantage point inside of a business, as we synthesize pain points, needs and business challenges across multiple business units. Contrast this with the visibility of prod-eng teams, who largely operate within their own scope, mandate and service area ownership. While these prod-eng teams have a very deep and specific understanding of their services and use cases, they have a limited knowledge of how other services or parts of the product work.
```

Comment:
```suggestion
What I love most about my job is that product designers cut across entire organizations. The success of the role is largely dependent on building trust and establishing true partnerships with key stakeholders. Design interfaces with growth marketing, dev rel, product, engineering, and customer success. This gives us a unique vantage point inside a business because we synthesize pain points, needs, and business challenges across multiple business units. Contrast this with the visibility of prod-eng teams, who largely operate within their own scope, mandate, and service area ownership. Although these prod-eng teams have a very deep and specific understanding of their services and use cases, they have a limited knowledge of how other services or parts of the product work.
```